washee_box/RaspberrySetupFiles/readme.md

```diff
@@ -0,0 +1,2 @@
+# this is how to set this stuff up
+run the setup.sh script
```

Comment:
I think it is better to have the description of the setup in the Washee_box README instead of a nested readme.

Also, how is the script run? Through ssh using bash in the raspberry pi? Does any directory work, or should it be run from somewhere specific? With or without sudo access?